docs/content/v0.20/examples.md

```diff
@@ -199,12 +216,43 @@ EOF
 To create a second WDS based on the hosting cluster, run the command:
 
 ```shell
-kflex create wds2 -t host -p kubestellar
+kflex create wds2 -t host
 ```
 
 where the `-t host` option specifies a control plane of type `host`.
 You can only create on control plane of type `host`.
 
+In this example, we use the helm chart method to install the kubestellat controller manager for the
+hosting cluster so that we can pass additional startup options.
+
+Label the `wds2` control plane as type `wds`:
+
+```shell 
+kubectl label cp wds2 kflex.kubestellar.io/cptype=wds
+```
+
+For this example, we use the `AppWrapper` custom resource defined in the
+[multi cluster app dispatcher](https://github.com/project-codeflare/multi-cluster-app-dispatcher)
+project.
+
+Install the AppWrapper CRD in the WDS and the WECs. Note that due to 
+[this issue](https://github.com/kubestellar/kubestellar/issues/1705) CRDs must be pre-installed 
+on the WDS and on the WECs when using API group filtering. For KubeStellar `0.20.0-rc1` this is required 
+when using a hosting cluster as WDS with a large number of API resources such as OpenShift.
+
+```shell
+clusters=(wds2 cluster1 cluster2);
+  for cluster in "${clusters[@]}"; do
+  kubectl --context ${cluster} apply -f https://raw.githubusercontent.com/project-codeflare/multi-cluster-app-dispatcher/v1.39.0/config/crd/bases/workload.codeflare.dev_appwrappers.yaml
+done
+```
+
+Apply the kubestellar controller-manager helm chart with the option to allow only delivery of objects with api group `workload.codeflare.dev`
+
+```shell
+helm upgrade --install -n wds2-system kubestellar oci://ghcr.io/kubestellar/kubestellar/controller-manager-chart --version 0.20.0-rc1 --set ControlPlaneName=wds2 --set APIGroups=workload.codeflare.dev
```

Comment:
I do not see an explicit kubeconfig context specification on this command line. I think we should stick with making that explicit. For the `helm` command this would be specified with the `--kube-context` command line flag.

Reply:
fixed